sdk/storage/Azure.Storage.Blobs.AspNetCore.DataProtection/tests/MockResponse.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Azure;
+using Azure.Core;
+
+namespace Microsoft.AspNetCore.DataProtection.Azure.Storage.Blob
+{
+    // Copied from https://github.com/Azure/azure-sdk-for-net/blob/43a178cbb60a50d9841b48170681b9187143e551/sdk/core/Azure.Core/tests/TestFramework/MockResponse.cs
+    public class MockResponse : Response
```

Comment:
Any reason we're not pulling it in as shared source?

Reply:
Leftovers from copying it from AspNetCore repo.